### DISCUSSION TITLE
What is the best styling libraries to use with Next.js?

### DISCUSSION THREAD

**Discussion by ayrtongomes** [vercel/next.js#18207] [Help]
Recently, I saw that [styled-components has support for SSR](https://styled-components.com/docs/advanced#nextjs).
I'm asking if it have any downsides(mainly related to SEO).
I'm currently using SCSS with CSS modules.

**Comment by jurajkrivda** [comment]
I use Rebass or Theme-UI

**Reply by jurajkrivda** [reply]
I used Theme-UI over Rebass because Theme-UI has better TS support. But now I use Chakra-UI.
